src/meta/src/rpc/service/mod.rs

```diff
@@ -25,28 +25,30 @@ use std::pin::Pin;
 use std::task::{Context, Poll};
 
 use futures::Stream;
-use risingwave_common::error::{tonic_err, RwError};
 use tokio::sync::mpsc::Receiver;
 
-/// `RwReceiverStream` is a wrapper around `tokio::sync::mpsc::Receiver` that implements Stream.
-/// `RwReceiverStream` is similar to `tokio_stream::wrappers::ReceiverStream`, but it maps Result<S,
-/// `RwError`> to Result<S, `tonic::Status`>.
-pub struct RwReceiverStream<S> {
-    inner: Receiver<Result<S, RwError>>,
+use crate::error::meta_error_to_tonic;
+use crate::MetaError;
+
+/// `MetaErrorReceiverStream` is a wrapper around `tokio::sync::mpsc::Receiver` that implements
+/// Stream. `MetaErrorReceiverStream` is similar to `tokio_stream::wrappers::ReceiverStream`, but it
+/// maps Result<S, `MetaError`> to Result<S, `tonic::Status`>.
+pub struct MetaErrorReceiverStream<S> {
```

Comment:
Seems we should keep the original name `Rw` here. 🤣

Reply:
🤣I will change it soon.